tempo_project/dimensions/subset/enduses_by_fuel_type.csv

```diff
@@ -0,0 +1,3 @@
+id,electricity_end_uses
+electricity_ev_dcfc,x
+electricity_ev_l1l2,x
```

Comment:
Redundant with all end uses? Or maybe this is better with regards to fuel_type ...